rllib/algorithms/algorithm_config.py

```diff
@@ -267,11 +279,11 @@ def __init__(self, algo_class=None):
         # `self.multi_agent()`
         self.policies = {DEFAULT_POLICY_ID: PolicySpec()}
         self.policy_map_capacity = 100
-        self.policy_map_cache = None
         self.policy_mapping_fn = (
             lambda aid, episode, worker, **kwargs: DEFAULT_POLICY_ID
         )
         self.policies_to_train = None
+        self.policies_swappable = False
```

Comment:
New setting for indicating: policies in map are state-swappable with each other.